src/client/datascience/interactive-window/interactiveWindow.ts

```diff
@@ -587,7 +593,19 @@ export class InteractiveWindow implements IInteractiveWindowLoadable {
                 [notebookCellData]
             );
         });
-        return notebookDocument.cellAt(notebookDocument.cellCount - 1);
+        const cell = notebookDocument.cellAt(notebookDocument.cellCount - 1);
+
+        // All cells added are immediately executed. Indicate that now
```

Comment:
This comment is incorrect. I wanted to mark cells as 'busy' but the Notebook API doesn't make that  easy. I would actually have to make them part of a CellExecution, but since we queue those up, I can't.